src/main/java/lessonfour/hometask/repository/impl/UserRepositoryImpl.java

```diff
@@ -0,0 +1,83 @@
+package lessonfour.hometask.repository.impl;
+
+import lessonfour.hometask.exception.DatabaseException;
+import lessonfour.hometask.model.User;
+import lessonfour.hometask.repository.UserRepository;
+import lessonfour.hometask.util.ConnectionUtil;
+
+import java.sql.PreparedStatement;
+import java.sql.Connection;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+
+public class UserRepositoryImpl implements UserRepository {
+    private static final int MIN_AFFECTED_ROWS_COUNT = 1;
+    private static final String ID = "id";
+    private static final String EMAIL = "email";
+    private static final String PHONE = "phone";
+    private static final String PASS = "password";
+
+    @Override
+    public User save(User user) {
+        String sqlRequest = "INSERT INTO users (email, phone, password) VALUES (?, ?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(sqlRequest, Statement.RETURN_GENERATED_KEYS)
+        ) {
+            preparedStatement.setString(1, user.getEmail()); // Добавил @Getter
```

Comment:
please use comments only in English. Also, this comment is redundant since it makes no sense